### ISSUE TITLE
No description on overmap

### ISSUE THREAD

**Issue by MarceloLaser** [Larwck/Graphical_Overmap#2]
When using either the latest version from this repository, or the 3rd of January version from CleverRaven, overmap tiles show no description on the right-hand side vertical bar. I am running build 10131 on Windows 10, screen resolution is 3840 x 2160. Overmap font sizes are all set to 16, as instructed.

EDIT: Further testing shows that the issue is due to font size 16x16x16, NOT the graphical overmap mod. Adding a solution to the mod's readme would be beneficial, since the causing font size is a requirement for the mod to work.

**Comment by Larwck** [comment]
Yeah I'm also getting this issue. Thanks for figuring out the cause. I'm unclear as to the solution. I'll ask for help in the discord.

**Comment by Larwck** [comment]
Apparently it's due to a change in the UI drawing, there's nothing I can do about it until that's fixed. Apologies. I'll make a note in the readme.
